source/proof_document_mock/lib/proof_document_mock.rb

```diff
@@ -0,0 +1,103 @@
+require 'bundler/setup' if !defined?(Bundler)
+require 'faraday'
+require 'json'
+require 'proofer'
+require 'retries'
+require_relative 'document_mock_client'
+require '/opt/ruby/lib/faraday_helper' if !defined?(IdentityIdpFunctions::FaradayHelper)
+require '/opt/ruby/lib/ssm_helper' if !defined?(IdentityIdpFunctions::SsmHelper)
+
+module IdentityIdpFunctions
+  class ProofDocumentMock
+    include IdentityIdpFunctions::FaradayHelper
+
+    def self.handle(event:, _context:, &callback_block)
```

Comment:
I don't think this can be `_context` because that is the specific name of the param that the lambda API passes in, we need to rubocop disable it like here: https://github.com/18F/identity-idp-functions/blob/master/source/proof_address/lib/proof_address.rb#L13

Reply:
AWS documentation: https://docs.aws.amazon.com/lambda/latest/dg/ruby-handler.html